cmd/juju/model/destroy.go

```diff
@@ -89,24 +92,33 @@ without delay waiting for each step to complete.
 Examples:
 
     juju destroy-model test
-    juju destroy-model -y mymodel
-    juju destroy-model -y mymodel --timeout 5m
-    juju destroy-model -y mymodel --destroy-storage
-    juju destroy-model -y mymodel --release-storage
-    juju destroy-model -y mymodel --force
-    juju destroy-model -y mymodel --force --no-wait
+    juju destroy-model --no-prompt mymodel
+    juju destroy-model --no-prompt mymodel --timeout 5m
+    juju destroy-model --no-prompt mymodel --destroy-storage
+    juju destroy-model --no-prompt mymodel --release-storage
+    juju destroy-model --no-prompt mymodel --force
+    juju destroy-model --no-prompt mymodel --force --no-wait
 
 See also:
     destroy-controller
 `
-var destroyIAASModelMsg = `
+
+var destroyModelMsg = `
 WARNING! This command will destroy the %q model.
-This includes all machines, applications, data and other resources.
 `[1:]
 
-var destroyCAASModelMsg = `
+var destroyIAASModelMsgWithDetails = `
+WARNING! This command will destroy the %q model.
+This includes %d machine(s), %d application(s), data and other resources.
+
+The following machines will be destroyed: %s
```

Comment:
I would suggest listing the applications that will be destroyed as well. Maybe consider just application, since machine Ids on aren't particularly meaningful.

However, I'm not sure what the difference between f.Id and f.InstanceId is so I could be wrong

Reply:
Agreed, but firstly we ween to change `api.ModelStatus` to get info about apps.